### PR TITLE
Test the GraphQL descriptions feature

### PR DESCRIPTION
**Pull request by ajhyndman**
This is required for #54.

This is also a prerequisite for #105.

@rafalp Again, let me know if I'm on the right track, or if you'd like me to expand on anything!

### CODE REVIEW

File: tests/test_descriptions.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    description = schema.get_type("Query").description
+    assert description == "A simple GraphQL schema which is well described."
+
+
+def test_object_fields_include_descriptions():
+    schema = make_executable_schema([query_definition, enum_definition])
+
+    description = schema.get_type("Query").fields.get("translate").description
+    assert (
+        description
```
Comment (rafalp) [inline tests/test_descriptions.py]: Is this formatted by black? Could you replace the line with `description == ("multiline string")` and see if its better?

File: tests/test_descriptions.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    description = schema.get_type("Language").description
+    assert description == "The set of languages supported by `translate`."
+
+
+def test_enum_values_include_descriptions():
+    schema = make_executable_schema(enum_definition)
+
+    for value in schema.get_type("Language").values.values():
+        assert isinstance(value.description, str)
```
Comment (rafalp) [inline tests/test_descriptions.py]: Could this be replaced with 1 assert explicitly testing one of items for having description?

Loops generate nice coverage, but they provide any feedback if they didn't loop over anything, which is bad. :(

File: tests/test_descriptions.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        description
+        == "Translates a string from a given language into a different language."
+    )
+
+
+def test_object_field_arguments_include_descriptions():
+    schema = make_executable_schema([query_definition, enum_definition])
+
+    translate_field = schema.get_type("Query").fields.get("translate")
+
```
Comment (rafalp) [inline tests/test_descriptions.py]: I would drop this blank line. Doesn't seem like it adds much to readability here imho.

File: tests/test_descriptions.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    for argument in translate_field.args.values():
+        assert isinstance(argument.description, str)
```
Comment (rafalp) [inline tests/test_descriptions.py]: Same feedback as for one above. Please replace loop with explicit assert.

**Comment by ajhyndman** [comment]
Thanks for the detailed feedback!

Sorry for the slow turnaround, I promise I don't mean to drag my feet.  Updating this just hasn't made it to the top of my todo list every day.

**Comment by rafalp** [comment]
Great work! Thanks!

**Pull request merged**